platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,66 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, MGLLightAnchorType) {
+    MGLLightAnchorTypeMap,
+    MGLLightAnchorTypeViewport
+};
+
+MGL_EXPORT
+@interface MGLLight : NSObject
+
+/**
+ `lightAnchorType` property.
+ */
+@property (nonatomic) MGLStyleValue<NSValue *> *lightAnchorType;
```

Comment:
The style specification was incorrect: mapbox/mapbox-gl-js#4743. Tracking a fix to this API in #9085.